### DISCUSSION TITLE
Compatibility issue with Alpine UI and livewire ^3.4.11

### DISCUSSION THREAD

**Discussion by Ahrengot** [livewire/livewire#8555] [7. Bugs (archived)] [answered]
### Livewire version

3.5.0

### Laravel version

10.47.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Click the toggle button to toggle the popover
2. Enter something in the input field
3. `Uncaught Could not find Livewire component in DOM tree` is thrown in the JS console and the popover breaks

### Repository or Wirebox URL

https://wirebox.app/b/xgog6

### Additional information

The issue appears to happen when Livewire (or Alpine?) is morphing the dom after the input is updated.

The panel directive `x-popover:panel` seems to be what's throwing the error. When removed, no exception is thrown after typing into the input, so the bug might be isolated to somewhere within: https://github.com/alpinejs/alpine/blob/1927cb1863d63c0502b2e66b1c3993bbed7e6cb0/packages/ui/src/popover.js#L148-L192 though I've had no luck debugging it.

Things I've tested:

1. The issue goes away when not using teleport. 
2. The issue does not affect the menu component in Alpine UI.

I've been trying different livewire version between 3.4.9 and 3.5. The issue is introduced in 3.4.11, so somewhere in this diff: https://github.com/livewire/livewire/compare/v3.4.10...v3.4.11 — Maybe the upgrade to alpine 3.13.9?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by jplhomer** [comment]
I'm having this same issue and haven't been able to track it down, either.

- Removing `teleport` also fixes it for me
- I'm also using a `menu` with an `anchor`
- I get the same error messages: "Cannot find Livewire component in DOM" and "x-anchor plugin..."

**Comment by jplhomer** [comment, marked as answer]
@Ahrengot I Found the suspect PR in Alpine, as well as a solution (in case you still hadn't found one like me): https://github.com/alpinejs/alpine/pull/4169

Wrapping the teleported contents with `div` resolves the issue for me 🎉

**Reply by Ahrengot** [reply]
Thank you @jplhomer!
